.github/workflows/build-test.yml

```diff
@@ -29,33 +29,62 @@ jobs:
       - name: Restore node_modules
         uses: OffchainLabs/actions/node-modules/restore@main
 
-      - uses: ./.github/actions/create-env-file
+      - name: Create .env file
+        uses: SpicyPizza/create-envfile@v1.3
         with:
-          NEXT_PUBLIC_INFURA_KEY: ${{ secrets.NEXT_PUBLIC_INFURA_KEY }}
+          envkey_NEXT_PUBLIC_INFURA_KEY: ${{ secrets.NEXT_PUBLIC_INFURA_KEY }}
+          directory: ./packages/arb-token-bridge-ui/
+          file_name: .env.local
 
       - name: Build
         run: yarn build
 
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: build-artifacts
+          path: |
+            ./packages/arb-token-bridge-ui/.env.local
+            ./packages/arb-token-bridge-ui/build
+            ./packages/token-bridge-sdk/dist
+            ./packages/use-wallet/dist
+
+  test-sdk:
+    name: "Test SDK"
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Restore node_modules
+        uses: OffchainLabs/actions/node-modules/restore@main
+
+      - name: Download build artifacts
```

Comment:
We could move it to an action (only for this repo)?

Reply:
Updated!